src/api/searchSdsDevice/tests/test_index.py

```diff
@@ -316,3 +316,80 @@ def test_filter_errors(params, error, status_code):
         )
     assert result["statusCode"] == status_code
     assert error in result["body"]
+
+
+@pytest.mark.integration
+@pytest.mark.parametrize(
+    "params, devices",
+    [
+        (
+            {
+                "nhs_as_client": "5NR",
+                "nhs_as_svc_ia": "urn:nhs:names:services:mm:PORX_IN090101UK31",
+            },
+            [
+                {"device_key": "P.AAA-CCC", "device_name": "device-name-a"},
+                {"device_key": "P.AAA-HHH", "device_name": "device-name-b"},
+            ],
+        ),
+    ],
+)
+def test_only_active_returned(params, devices):
```

Comment:
Sorry, should have said can we recreate this test in the searchSdsEndpoint tests. It should be almost identical